Formula/geogram.rb

```diff
@@ -0,0 +1,25 @@
+class Geogram < Formula
+  desc "Programming library of geometric algorithms"
+  homepage "http://alice.loria.fr/software/geogram/doc/html/index.html"
+  url "https://gforge.inria.fr/frs/download.php/file/37635/geogram_1.6.6.tar.gz"
+  sha256 "08211b1d6f21e14701e3fd5c79adbe331cdf66b8af84efdb54cd7048244691b5"
+  depends_on "cmake" => :build
+
+  def install
+    # don't write directly into /usr/local/lib/cmake/modules
+    inreplace "CMakeLists.txt", "lib/cmake/modules", "#{share}/cmake/Modules"
+    # workaround for undefined _mm_set_pd1 in clang
+    inreplace "src/lib/geogram/numerics/predicates.cpp", "_mm_set_pd1", "_mm_set1_pd"
```

Comment:
This is something we're going to need upstream to handle, rather than fixing it ourselves.

Reply:
Do I only need to try to get upstream to handle the `_mm_set_pd1` issue, or to also make the CMake module location configure-time configurable?

Reply:
Both would be nice 😸, but the `_mm_set_pd1` issue is the one I was referencing here.

Reply:
The `_mm_set_pd1` issue should be fixed upstream in the next release: https://lists.gforge.inria.fr/pipermail/geogram-users/2018-August/000158.html